windows-agent/internal/proservices/landscape/executor_test.go

```diff
@@ -205,7 +198,13 @@ func TestInstall(t *testing.T) {
 
 			if tc.appxDoesNotExist || tc.sendRootfsURL != "" {
 				// WSLMock Install only accepts ubuntu-22.04
-				settings.name = wsltestutils.RandomDistroName(t)
+				if tc.distroName == nil {
+					settings.name = wsltestutils.RandomDistroName(t)
+				} else {
+					n, ok := tc.distroName.(string)
+					require.True(t, ok, "Setup: tc.distroName should be a string")
```

Comment:
yeah, that would simplify all this to use the other pattern for empty/default I explained below.